fontconfig/plan.sh

```diff
@@ -27,12 +51,9 @@ do_prepare() {
 }
 
 do_build() {
-  export PKG_CONFIG_PATH="$(pkg_path_for freetype)/lib/pkgconfig:$(pkg_path_for expat)/lib/pkgconfig"
-
-  ./configure --sysconfdir=${pkg_prefix}/etc \
-              --prefix=${pkg_prefix} \
+  ./configure --sysconfdir="$pkg_prefix/etc" \
+              --prefix="$pkg_prefix" \
               --disable-static \
-              --mandir=${pkg_prefix}/man
+              --mandir="$pkg_prefix/man"
```

Comment:
No it didn't. The only reason you need to use curly braces is if there's not a separator, so if it was like `${pkg_prefix}man`. They aren't necessary and if shellcheck doesn't complain about them, I don't think we should use them.